alphagov/paas-bootstrap
# Remove access from Jamie and Malcolm

What
----

Remove access to concourse for Jamie and Malcolm, and remove their GPG keys from those allowed to sign merge commits, as they have left the team

How to review
-------------

* Set a dev concourse environment to use this branch
* Upload the trusted people file using the `remove-feb-leavers` branch: `gds aws paas-dev-admin -- make <dev-env-name> upload-paas-trusted-people PAAS_TRUSTED_PEOPLE_BRANCH=remove-feb-leavers`
* run the `create-bosh-concourse` pipeline and confirm that access for Jamie and Malcolm has been removed.
* Ensure that the GPG keys belonging to Jamie van Dyke and Malcolm Saunders have been removed from `.gpg-id`


Who can review
--------------

Existing PaaS team members with access to dev environments

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
